### ISSUE TITLE
Google Colab error

### ISSUE THREAD

**Issue by kayanjaandrew** [lululxvi/deepxde#1486]
I am a newbie to DeepXDE. I am applying deepxde and PyTorch backend (from deepxde.backend import torch) to solve an 
ODE(SIR model) inverse problem. When I  use my computer CPU to train the model, everything works fine when it comes 
 to checking the parameter estimates using :
estimated_beta = beta.detach().numpy()
estimated_alpha = alpha.detach().numpy()
But when i transfer the program to Google Colab,  i get the error "'RefVariable' object has no attribute 'detach'"

**Comment by kayanjaandrew** [comment]
Solved the problem by setting the right backend.

**Comment by AmirNoori68** [comment]
@kayanjaandrew , how to change the backend to "pytorch" in google colab?!
We do the following installation first: 

"pip install DeepXDE
import deepxde as dde
"
 and then what?

Thanks
